src/cards/resolvers.ts

```diff
@@ -1,4 +1,17 @@
-import { allCards } from '../util'
+import { allCards, getCardForDay } from '../util'
+
+// Not sure if this works on the day a new set gets released since it would change the amount of cards
+// and change the index for yesterday's guess
+async function pastCard(daysBack: number) {
+  if(daysBack == 0) return null
```

Comment:
If we're going to prevent getting today's, should we also prevent getting future cards?

Reply:
I just took the absolute value so it is always going backwards. so if you ask for 2 days back or -2 days back you will get 2 days back 🤷 this way you can't really misuse the endpoint

Reply:
https://github.com/Billzabob/lordle_backend/pull/1/files#diff-b9525fe0c2430986fb98dc2b17b128f2a96bcd19bc419675af32db56867d31f5R25

Reply:
thats v clever

Reply:
Oh nice!